src/analyticsdf/analyticsdataframe.py

```diff
@@ -335,3 +335,22 @@ def update_response_poly_categorical(self, predictor_name: str = None, betas: di
                 self.response_vector = numeric_vals[predictor_name]
             else:
                 self.response_vector += numeric_vals[predictor_name]
+
+    def update_predictor_bernoulli(self, predictor_name_list,
+                               prob):
+        """Update the predictors of the instance to bernoulli distributed.
+
+        Args:
+            predictor_name_list:
+                A list of predictor names in the initial AnalyticsDataframe.
+            prob:
+                float or array_like of floats, probability of success
+           
+        Raises:
```

Comment:
I think there is no exception being raised in this function, please use the 'checking' function if you want the exception raised as expected here.